README.md

```diff
@@ -49,7 +49,7 @@ This will produce a single file (Pax.exe), called an *assembly* in .NET jargon.
 # Writing for Pax
 Packet processers using Pax can be written in any [.NET language](https://en.wikipedia.org/wiki/List_of_CLI_languages).
 They use Pax's API and define one or more functions that handle incoming packets.
-The [examples](https://github.com/niksu/pax/tree/master/examples) included with Pax could help get you going.
+The [examples](examples) included with Pax could help get you going.
```

Comment:
It looks great to me -- factoring out the redundant info is good, and it 
shouldn't impact usability in this case.

On 2016-07-06 15:13, TMVector wrote:
> In README.md [1]:
> 
>> @@ -49,7 +49,7 @@ This will produce a single file (Pax.exe), called
> an *assembly* in .NET jargon.
>> # Writing for Pax
>> Packet processers using Pax can be written in any [.NET
> language](https://en.wikipedia.org/wiki/List_of_CLI_languages).
>> They use Pax's API and define one or more functions that handle
> incoming packets.
>> -The [examples](https://github.com/niksu/pax/tree/master/examples)
> included with Pax could help get you going.
>> +The [examples](examples) included with Pax could help get you
> going.
> 
> What do you think of this @niksu [2]? I checked it works, and it's
> more portable, but I feel like the links are slower?
> 
> --
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub [3], or mute the
> thread [4].
> 
> 	*
> 
> Links:
> ------
> [1] https://github.com/niksu/pax/pull/3#discussion_r69737670
> [2] https://github.com/niksu
> [3]
> https://github.com/niksu/pax/pull/3/files/5b3946dfadd9146af0c692313023c6894b758922..99626d3e8070bfe410fdffa4f7719e36451a8f64#r69737670
> [4]
> https://github.com/notifications/unsubscribe/ABDTTuxq1kQVT5dWNVL8_PNp3fwUfopFks5qS7f9gaJpZM4JFDoJ

-- 
http://www.cl.cam.ac.uk/~ns441/